src/uu/chroot/src/chroot.rs

```diff
@@ -48,6 +49,20 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         None => return Err(ChrootError::MissingNewRoot.into()),
     };
 
+    let skip_chdir = matches.contains_id(options::SKIP_CHDIR);
+    // We are resolving the path in case it is a symlink or /. or /../
+    if skip_chdir
+        && canonicalize(newroot, MissingHandling::Normal, ResolveMode::Logical)
+            .unwrap()
+            .to_str()
+            != Some("/")
+    {
+        return Err(USimpleError::new(
+        125,
+        "option --skip-chdir only permitted if NEWROOT is old '/'\nTry 'chroot --help' for more information.",
```

Comment:
We have [`UUsageError`](https://docs.rs/uucore/latest/uucore/error/struct.UUsageError.html) for the "Try 'util --help'" message, which will also use correct util name automatically.

Reply:
Thanks :)